recipes/dlatk/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "dlatk" %}
+{% set version = "1.2.3" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/dlatk-{{ version }}.tar.gz
+  sha256: fc4c8f12c597623facb5100e6d836a191459655938c83ccc352df377bd07cebd
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - numpy
```

Comment:
This probably isn't needed
```suggestion
```

Reply:
It looks like you actually do need `numpy` as the package pointlessly includes it in `setup_requires`: https://github.com/dlatk/dlatk/blob/87034d79b1f8255ac88e1bbac003e8936e3e2772/setup.py#L149-L151

Reply:
yep added it back, thanks!